README.md

```diff
@@ -664,7 +646,8 @@ Now that we’ve gone over how to [use the `over_react` package in your project]
 the [anatomy of a component](#anatomy-of-an-overreact-component) and the [DOM components](#dom-components-and-props)
 that you get for free from OverReact, you're ready to start building your own custom React UI components.
 
-1. Start with one of the [component boilerplate templates](#component-boilerplate-templates) below.
+1. Start with one of the [component boilerplate templates](#component-boilerplate-templates) below 
+(Or, use OverReact's [code snippets for Intellij and Vs Code](snippets/README.md)).
```

Comment:
this relative link won't work when the readme is published, for instance, to `pub.dartlang.org`.  It needs to be an absolute link to that readme.

Reply:
Same thing on line 672